application/src/Api/Adapter/ApiResourceAdapter.php

```diff
@@ -17,7 +17,7 @@ public function getResourceName()
 
     public function getRepresentationClass()
     {
-        return 'Omeka\Api\Representation\ApiResourceRepresentation';
+        return \Omeka\Api\Representation\ApiResourceRepresentation::class;
```

Comment:
Official guidance on class names as strings is to omit the leading backslash since class names in strings are always presumed to be fully-qualified.

If we have places where we _are_ including the leading backslash in a string, I'd consider _that_ something we should fix, but not in this direction.

Reply:
Wait, I misread these I guess. Of course using `::class` is fine. I'm sure I'd seen them as just backslashes in the string but I must be imagining things.

This is fine.

Reply:
Use a class is simpler to use in the editor.